android-json-form-wizard/src/main/java/com/vijay/jsonwizard/widgets/RepeatingGroupFactory.java

```diff
@@ -221,22 +255,33 @@ private void retrieveRepeatingGroupCountFromRemoteReferenceEditText(final @NonNu
     }
 
     @VisibleForTesting
-    protected void setGlobalLayoutListener(@NonNull final View rootLayout, @NonNull final MaterialEditText referenceEditText, final int remoteReferenceValue) {
+    protected void setGlobalLayoutListener(@NonNull final View rootLayout,
```

Comment:
Any reason for this because this change might break other people's code.

Reply:
https://github.com/OpenSRP/opensrp-client-native-form/pull/476/files/877163f20bacfba6fbfb6eeb77f053c9813ec04f#r484300334